configs/ote_custom_classification/efficientnet_b0/template.yaml

```diff
@@ -4,7 +4,7 @@ name: EfficientNet-B0
 task_type: CLASSIFICATION
 task_family: VISION
 instantiation: "CLASS"
-summary: Custom Image Classification EfficientNet-B0
+summary: Provides better performance on large datasets, but may be not so stable in case of small amount of training data.
```

Comment:
Do we keep efficientnetb0 for v1.1?  About a small amount. I would consider providing the lower boundary for this small amount (maybe 300-400?)

Reply:
I guess no. At the moment procedure of deleting model templates is not well established, so I can't do it right now. I don't know who will read this description and can we add any exact numbers here. @Ilya-Krylov what do you think?

Reply:
I think we should discuss it first